scripts/battlefields/Temenos/central_temenos_basement.lua

```diff
@@ -54,7 +55,10 @@ content.groups =
                     end
                 end)
 
-                -- TODO: Aern should drop ABC = 1 + math.min(3, mob:getLocalVar("AERN_RERAISES")) (before the reraise)
+                mob:addListener("ITEM_DROPS", "ITEM_DROPS_AERN", function(mobArg, loot)
+                    local quantity = math.min(3, mob:getLocalVar("AERN_RERAISES"))
```

Comment:
Comment said `1 + math.min(3, mob:getLocalVar("AERN_RERAISES"))`
was that wrong?

Reply:
Correct. I wasn't sure if this would run before or after the bar is incremented. One is also coming from the drop id set on the mob pool